src/Microsoft.Azure.WebJobs.Logging.ApplicationInsights/ApplicationInsightsLoggerOptions.cs

```diff
@@ -194,6 +200,16 @@ public string Format()
                 };
             }
 
+            JObject tokenCredentialOptions = null;
+            if (TokenCredentialOptions != null)
+            {
+                tokenCredentialOptions = new JObject
```

Comment:
Do you think it's worth running this through sanitization incase a customer puts a full connection string in one of these properties?

Reply:
I have removed the Authorization property. ClientId is masked, I just need this for troubleshooting.